### DISCUSSION TITLE
No more temp info on the "phpSysInfo" page

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#806] [Q&A]
Hello,

Since yesterday I have two new issues:

-I don't have the temp info on the "Sys info" page.
-I always have this message on the main page: 

> Your system is currently processing a backlog of audio. This can take several hours before normal functionality of your BirdNET-Pi resumes.

Thank you for any suggestions.

**Comment by becausebirds** [comment]
I've been seeing the currently processing message a lot recently too on both my existing station and a new one I just set up.
